winui/Masked-TextBox/Events.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Events in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the events in the MaskedTextBox (SfMaskedTextBox) control.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# Events in WinUI MaskedTextBox
+
+This section explains the events available in the WinUI MaskedTextBox control.
```

Comment:
@BhaskarSuresh22 
This section provides information about the events available in the WinUI MaskedTextBox control.

## ValueChanging Event
The ValueChanging event occurs when the Value property is about to change in the MaskedTextBox control. The MaskedTextBoxValueChangingEventArgs object provides the following properties:

NewValue: Gets the current value of the MaskedTextBox control.
OldValue: Gets the previous value of the MaskedTextBox control.
IsValid: Gets or sets a boolean value indicating whether the input is considered valid based on the mask completion.
Cancel: Gets or sets a value indicating whether the event should be canceled.

private void MaskedTextBox_ValueChanging(object sender, MaskedTextBoxValueChangingEventArgs e)
{
    // Access the new and old values
    string newValue = e.NewValue;
    string oldValue = e.OldValue;

    // Check the validity of the input
    bool isValid = e.IsValid;

    // Cancel the event if needed
    if (newValue == "1234")
    {
        e.Cancel = true;
    }

    // Perform additional actions based on the value changing
    // ...
}

Reply:
@Manivannan-E, Done the changes.